examples/phpstan-full.neon

```diff
@@ -26,15 +26,6 @@ parameters:
     ignoreErrors:
         # Uses func_get_args()
         - '#^Function apply_filters(_ref_array)? invoked with [34567] parameters, 2 required\.$#'
```

Comment:
This could also go.

Reply:
I'm not sure about `apply_filters_ref_array`, that's why I did not remove it. And yes, `apply_filters` could be removed.